app/assets/javascripts/application.js.erb

```diff
@@ -228,3 +229,11 @@ var HelperModule = (function(){
 
   return helpers;
 })();
+
+(function() {
```

Comment:
What does this do, render highlight on code syntax? Can you maybe just add a code comment, so we will know what's it for?